src/node_dotenv.cc

```diff
@@ -267,4 +250,13 @@ void Dotenv::AssignNodeOptionsIfAvailable(std::string* node_options) const {
   }
 }
 
+std::optional<std::string> Dotenv::GetValue(const std::string_view key) const {
```

Comment:
This can return std::string_view since the underlying store will not be disposed.

```suggestion
std::optional<std::string_view> Dotenv::GetValue(const std::string_view key) const {
```

Reply:
Done